### DISCUSSION TITLE
Is there a table of the summary descriptions?

### DISCUSSION THREAD

**Discussion by gquiring** [Pirate-Weather/pirateweather#206] [General]
I have limited space for displaying the weather summary, I need to map the descriptions to 8 or less characters for the display I use.  Most of the API's use a weather code and I could map a description to the code.  Where can I get a list of the summary descriptions?

**Comment by alexander0042** [comment]
Great question! I'd suggest using the "icon" block, which provides a simple summary of the forecast weather conditions. The list of possible icons is here: `https://docs.pirateweather.net/en/latest/API/#icon`. I'd also suggest adding logic for a few other possible types per issue #43; however, they are not implemented yet.
